### DISCUSSION TITLE
Can you customize the display colors of the Symbol Tree?

### DISCUSSION THREAD

**Discussion by matcool** [NationalSecurityAgency/ghidra#3038] [Q&A]
recently i've been messing with my ghidra colors and managed to customize the listing and decompiler view quite well, however the symbol tree somehow got its colors messed up in the process, making it quite hard to read the text
![image](https://user-images.githubusercontent.com/26722564/118417457-82bb7f00-b68a-11eb-93ac-6b67ea96d3f3.png)
i've triple checked the config and nowhere did i see where you can change the colors in the symbol tree, so i was wondering if it was possible to change it

**Comment by dragonmacher** [comment]
The Symbol Tree picks colors via the `SymbolInspector`.    There are no direct options settings for the Symbol Tree specifically.    The `SymbolInspector` uses the settings found under `Listing Display` in the Tool Options.     Changing the values therein should allow you to update the Symbol Tree.    

The easiest thing to do would be to use the `Restore Defaults` button on that options panel.
